recipes/sciphi/v0.1.1/meta.yaml

```diff
@@ -21,15 +21,15 @@ requirements:
     - llvm # [osx]
     - libcxx # [osx]
     - cmake >=3.8
-    - boost #{ CONDA_BOOST }}*
-    - zlib #{ CONDA_ZLIB }}*
+    - boost {{ CONDA_BOOST }}*
+    - zlib {{ CONDA_ZLIB }}*
     - dlib
```

Comment:
@bioconda/core 

Hi, I was wondering if you could tell me how to be able to use the dlib (http://dlib.net/) header files. I know there is a dlib in conda-forge, however, it is not installing the header files. Since I am not allowed to supply more than one source entry I am a little lost.

Any help is much appreciated.

Many  thanks!